apis/mysql/v1alpha1/user_types.go

```diff
@@ -45,6 +45,19 @@ type UserParameters struct {
 	// See https://dev.mysql.com/doc/refman/8.0/en/user-resources.html
 	// +optional
 	ResourceOptions *ResourceOptions `json:"resourceOptions,omitempty"`
+
+	// AuthPlugin sets the mysql authentication plugin, defaults to mysql_native_password
+	// +optional
+	// +kubebuilder:validation:Pattern:=^([a-z]+_)+[a-z]+$
+	AuthPlugin *string `json:"authPlugin,omitempty" default:"mysql_native_password"`
```

Comment:
I think AuthPlugin got in here accidentally?

Reply:
Yes it did, thanks for pointing that out

Reply:
Fixed by https://github.com/crossplane-contrib/provider-sql/pull/135/commits/d1320e0dd9b9e8d73973589e6ee21c5125735c95